html2markdown.py

```diff
@@ -233,7 +233,6 @@ def _markdownify(tag, _listType=None, _blockQuote=False, _listIndex=1):
 				lines.pop()
 			for i,line in enumerate(lines):
```

Comment:
This loop doesn't do anything. I can't remember if `lines[i] = line.replace(...)` is still necessary, but it's currently not being assigned to `lines[i]`.